bigbluebutton-html5/client/main.html

```diff
@@ -1,5 +1,5 @@
 <head>
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
```

Comment:
Just curious - this should prevent users from zooming in Safari on iOS, right?
Does it work?
As far as I know Apple started ignoring these properties starting from iOS 10.